lms/djangoapps/courseware/tests/test_views.py

```diff
@@ -1345,17 +1342,17 @@ def test_progress_queries_paced_courses(self, self_paced, self_paced_enabled):
         """Test that query counts remain the same for self-paced and instructor-paced courses."""
         SelfPacedConfiguration(enabled=self_paced_enabled).save()
         self.setup_course(self_paced=self_paced)
-        with self.assertNumQueries(39), check_mongo_calls(4):
+        with self.assertNumQueries(41), check_mongo_calls(4):
             self._get_progress_page()
 
     def test_progress_queries(self):
         self.setup_course()
-        with self.assertNumQueries(39), check_mongo_calls(4):
+        with self.assertNumQueries(41), check_mongo_calls(4):
             self._get_progress_page()
 
         # subsequent accesses to the progress page require fewer queries.
         for _ in range(2):
-            with self.assertNumQueries(22), check_mongo_calls(4):
+            with self.assertNumQueries(24), check_mongo_calls(4):
                 self._get_progress_page()
```

Comment:
2 extra queries doesn't seem unreasonable to me - if this is too difficult to fix I wouldn't worry about it.